sys/dev/drm/freebsd/drm_os_freebsd.c

```diff
@@ -311,46 +311,9 @@ drm_vmap_find(void *handle)
 }
 
 static int
-drm_cdev_pager_fault(vm_object_t vm_obj, vm_ooffset_t offset, int prot,
-    vm_page_t *mres)
+drm_cdev_pager_fault(vm_object_t vm_obj __unused, vm_ooffset_t offset __unused,
+    int prot __unused, vm_page_t *mres __unused)
 {
-	struct vm_area_struct *vmap;
-
-	vmap = drm_vmap_find(vm_obj->handle);
-
-	MPASS(vmap != NULL);
-	MPASS(vmap->vm_private_data == vm_obj->handle);
-
-	if (likely(vmap->vm_ops != NULL && offset < vmap->vm_len)) {
```

Comment:
Hmm, how is it known to be dead?  Is vmap->vm_ops always NULL or something?

Reply:
Because this function is only used for `OBJT_DEVICE` objects, and there are no DRM objects of that type. I could rip out even more code, but my aim was just to remove that call to drm_vmap_find().

Reply:
Ok, wasn't obvious to me from the context.  Maybe allude to that in the commit log?

Reply:
Done.